grid_map_pcl/include/grid_map_pcl/PclLoaderParameters.hpp

```diff
@@ -47,6 +47,11 @@ class PclLoaderParameters {
     double resolution_ = 0.1;
     unsigned int minCloudPointsPerCell_ = 2;
```

Comment:
`useMaxHeightAsCellElevation_` in `ClustExtractionParameters` can be deleted, right?

Reply:
Fixed in https://github.com/ANYbotics/grid_map/pull/329/commits/efc2f165c6975ecf402bfa27a47400fc09bc8d6d